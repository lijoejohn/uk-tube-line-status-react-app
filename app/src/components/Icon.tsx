type SVGProps = React.SVGProps<SVGSVGElement>;

function Icon({ path, ...props }: SVGProps): JSX.Element {
  return (
    <svg
      role="img"
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fillRule="evenodd"
      clipRule="evenodd"
      {...props}>
      <path d={path} />
    </svg>
  );
}

export default Icon;
