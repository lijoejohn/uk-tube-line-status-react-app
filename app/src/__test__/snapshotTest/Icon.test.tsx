import * as renderer from 'react-test-renderer';
import Icon from '../../components/Icon';

describe('Icon Component', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Icon />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
