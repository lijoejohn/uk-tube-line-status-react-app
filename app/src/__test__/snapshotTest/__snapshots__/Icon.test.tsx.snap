// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Component should match the snapshot 1`] = `
<svg
  clipRule="evenodd"
  fillRule="evenodd"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <path />
</svg>
`;
